### PR TITLE
[2.0.x] Disabled test_exact_sliced_queryset_limit_one_offset on Oracle.

### PR DESCRIPTION
**Pull request by felixxm**
This required slicing on subquery with equal operator.

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: tests/lookup/tests.py [hunk 1]
```diff
@@ -854,6 +855,7 @@ def test_exact_sliced_queryset_limit_one(self):
             [self.a1, self.a2, self.a3, self.a4]
         )
 
+    @skipIf(connection.vendor == 'oracle', "Oracle doesn't support sliced subqueries with equal operator")
```
Comment (timgraham) [inline tests/lookup/tests.py]: You could add "until #28670" to that sentence.

**Comment by felixxm** [comment]
Merged in 3e7adb62bf2435ed82995a015705ddfa7d0f5a35.

**Pull request closed without merging**